scripts/inc.compatibility.sh

```diff
@@ -263,15 +263,13 @@ webviewstock"
 
 api23hack(){
   if [ "$API" -ge "23" ]; then
-    gappscore="$gappscore
-packageinstallergoogle"
     gappspico="$gappspico
 googletts"
     gappsmini="$gappsmini
 calculatorgoogle"
     gappsstock="$gappsstock
-contactsgoogle"
-#dialergoogle"
```

Comment:
Can you keep the #dialergoogle line there?
The idea is that it could be easily activated in the future when we will have the issues with the dialer sorted out.